pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -29,6 +29,9 @@ const (
 
 	// MachineClusterLabelName is the label set on machines linked to a cluster.
 	MachineClusterLabelName = "cluster.k8s.io/cluster-name"
+
+        // ExcludeNodeDrainingAnnotation annotation explicitly skips node draining if set
+        ExcludeNodeDrainingAnnotation = "machine.cluster.sigs.k8s.io/exclude-node-draining"
```

Comment:
Should I make this look more like cluster.k8s.io here?  I think it doesn't really matter and this will be one less thing to migrate.

Reply:
It should probably be namespaced under `cluster.k8s.io` since there are still ongoing discussions related to the namespacing of sig-related projects in #api-review in Kubernetes slack, and they will be spilling over to SIG Architecture.

Reply:
v1alpha1 should be cluster.k8s.io

Reply:
done.